<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-04-21T17:45:36 EDT" name="Suite" finished-at="2025-04-21T17:45:44 EDT" duration-ms="8067">
    <groups>
      <group name="ErrorHandling">
        <method signature="ErrorValidations.inValidEmailFormValidation()[pri:0, instance:com.automation.selenium_automation.tests.ErrorValidations@565f390]" name="inValidEmailFormValidation" class="com.automation.selenium_automation.tests.ErrorValidations"/>
        <method signature="ErrorValidations.loginErrorMessageValidation()[pri:0, instance:com.automation.selenium_automation.tests.ErrorValidations@565f390]" name="loginErrorMessageValidation" class="com.automation.selenium_automation.tests.ErrorValidations"/>
        <method signature="ErrorValidations.orderErrorValidation()[pri:0, instance:com.automation.selenium_automation.tests.ErrorValidations@565f390]" name="orderErrorValidation" class="com.automation.selenium_automation.tests.ErrorValidations"/>
      </group> <!-- ErrorHandling -->
    </groups>
    <test started-at="2025-04-21T17:45:36 EDT" name="Error Validation" finished-at="2025-04-21T17:45:44 EDT" duration-ms="8067">
      <class name="com.automation.selenium_automation.tests.ErrorValidations">
        <test-method is-config="true" signature="launchApp()[pri:0, instance:com.automation.selenium_automation.tests.ErrorValidations@565f390]" started-at="2025-04-21T17:45:36 EDT" name="launchApp" finished-at="2025-04-21T17:45:41 EDT" duration-ms="4752" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method is-config="true" signature="launchApp()[pri:0, instance:com.automation.selenium_automation.tests.ErrorValidations@565f390]" started-at="2025-04-21T17:45:36 EDT" name="launchApp" finished-at="2025-04-21T17:45:41 EDT" duration-ms="4764" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method is-config="true" signature="launchApp()[pri:0, instance:com.automation.selenium_automation.tests.ErrorValidations@565f390]" started-at="2025-04-21T17:45:36 EDT" name="launchApp" finished-at="2025-04-21T17:45:42 EDT" duration-ms="5865" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method signature="inValidEmailFormValidation()[pri:0, instance:com.automation.selenium_automation.tests.ErrorValidations@565f390]" started-at="2025-04-21T17:45:41 EDT" name="inValidEmailFormValidation" finished-at="2025-04-21T17:45:41 EDT" duration-ms="208" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- inValidEmailFormValidation -->
        <test-method signature="orderErrorValidation()[pri:0, instance:com.automation.selenium_automation.tests.ErrorValidations@565f390]" started-at="2025-04-21T17:45:41 EDT" name="orderErrorValidation" finished-at="2025-04-21T17:45:44 EDT" duration-ms="3147" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- orderErrorValidation -->
        <test-method is-config="true" signature="quitWebsite()[pri:0, instance:com.automation.selenium_automation.tests.ErrorValidations@565f390]" started-at="2025-04-21T17:45:41 EDT" name="quitWebsite" finished-at="2025-04-21T17:45:41 EDT" duration-ms="140" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitWebsite -->
        <test-method signature="loginErrorMessageValidation()[pri:0, instance:com.automation.selenium_automation.tests.ErrorValidations@565f390]" started-at="2025-04-21T17:45:42 EDT" name="loginErrorMessageValidation" finished-at="2025-04-21T17:45:43 EDT" duration-ms="709" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect banana or password.] but found [Incorrect email or password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect banana or password.] but found [Incorrect email or password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.automation.selenium_automation.tests.ErrorValidations.loginErrorMessageValidation(ErrorValidations.java:24)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginErrorMessageValidation -->
        <test-method is-config="true" signature="quitWebsite()[pri:0, instance:com.automation.selenium_automation.tests.ErrorValidations@565f390]" started-at="2025-04-21T17:45:43 EDT" name="quitWebsite" finished-at="2025-04-21T17:45:43 EDT" duration-ms="103" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitWebsite -->
        <test-method is-config="true" signature="quitWebsite()[pri:0, instance:com.automation.selenium_automation.tests.ErrorValidations@565f390]" started-at="2025-04-21T17:45:44 EDT" name="quitWebsite" finished-at="2025-04-21T17:45:44 EDT" duration-ms="122" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitWebsite -->
      </class> <!-- com.automation.selenium_automation.tests.ErrorValidations -->
    </test> <!-- Error Validation -->
  </suite> <!-- Suite -->
</testng-results>
